modules/nbde-tang-server-operator-identifying-url-cli.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * security/nbde_tang_server_operator/nbde-tang-server-operator-identifying-url.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="identifying-url-nbde-tang-server-operator-using-cli_{context}"]
+= Identifying URL of the NBDE Tang Server Operator using CLI
+
+You can identify URLs of Tang servers deployed with the NBDE Tang Server Operator from the OperatorHub using the CLI. After you identify the URLs, you use the `clevis luks bind` command on your clients containing LUKS-encrypted volumes you want to unlock automatically by keys advertised by the Tang servers. See the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption_security-hardening#configuring-manual-enrollment-of-volumes-using-clevis_configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption[Configuring manual enrollment of LUKS-encrypted volumes] section in the RHEL 9 Security hardening document for detailed steps describing the configuration of clients with Clevis.
+
+.Prerequisites
+
+* You must have `admin` privileges.
+* You have installed the OpenShift CLI (`oc`).
+* You deployed a Tang server using the NBDE Tang Server Operator on your OpenShift cluster.
+
+.Procedure
+
+. List details about your Tang server, for example:
++
+[source,terminal]
+----
+$ oc -n nbde describe tangserver
+…
+Spec:
+…
+Status:
+  Ready:                 1
+  Running:               1
+  Service External URL:  http://34.28.173.205:7500/adv
+  Tang Server Error:     No
+Events:
+…
+----
+
+. Use the value of the `Service External URL:` item without the `/adv` part. In this example, the URL of the Tang server is `http://34.28.173.205:7500`.
```

Comment:
See the doc guidelines on [Example URLs](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#example-urls).
```suggestion
. Use the value of the `Service External URL:` item without the `/adv` part. In this example, the URL of the Tang server is `\http://34.28.173.205:7500`.
```